_drafts/EH-tidyx.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Community Highlight: Ellis Hughes's TidyX Screencast!"
+author: Lauren Wolfe
+categories:
+  - Community
+  - Technical
+tags: 
+  - highlight
+  - tidyverse
+  - R
+---
+
+As we've been reassessing our work here at The Coop, we have found that there is a need to facilitate more interaction between the people that make up our community. It makes sense - we no longer have easy access to a quick catch up in the hallways or over lunch on campus and many of us have settled into our new socially-distant lifestyle. This is why The Coop Blog is going to shift some focus back onto our community by highlighting members and their data related work. For our first community highlight we're showcasing Ellis and his TidyX screencast!
```

Comment:
didn't we decide it was "the Coop"?

can we also include a link to TidyX here in the first paragraph?